cdap-ui/app/cdap/components/DataPrep/DataPrepTable/index.js

```diff
@@ -312,20 +312,13 @@ export default class DataPrepTable extends Component {
       return (
         <div className="dataprep-table empty">
           <div>
-            <h5 className="text-xs-center">Please create a workspace to wrangle data</h5>
-            <div className="button-container text-xs-center">
-              <button
-                className="btn btn-primary"
-                onClick={this.openCreateWorkspaceModal}
-              >
-                Create
-              </button>
-            </div>
+            <h5 className="text-xs-center">Please select or upload a file to wrangle data</h5>
           </div>
         </div>
       );
     }
 
+    // FIXME: Not sure if this is possible now.
```

Comment:
Yeah not sure what should we show if we don't have any data. Didn't want to randomly change that behavior. So the FIXME.